init_configure_commands_tut.md

```diff
@@ -0,0 +1,156 @@
+<a href="https://aws-amplify.github.io/" target="_blank">
+    <img src="https://s3.amazonaws.com/aws-mobile-hub-images/aws-amplify-logo.png" alt="AWS Amplify" width="550" >
+</a>
+
+# AWS Amplify CLI initialization & configuration tutorial
+
+The AWS Amplify CLI helps you set up an AWS IAM user, set up an AWS profile on your local system and the initialize Amplify powered apps with all your defined configurations. We'll also walk through steps as to how you could share the Amplify generated backend configuration between other frontend projects and re-use the same backend infrastructure in-between multiple platforms/users.
+
+In this tutorial, we'll go walk through setting up an Amplify enabled project and go in detail as to what happens behind the scenes when doing so.
+
+# Command usage
+
+## amplify configure
+```bash
+$ amplify configure
+```
+The `amplify configure` command is a one-time setup step which helps you with the following:
+  - Signing up and signing in into AWS
+  - Setting up an IAM user with the appropriate policies for the CLI to deploy AWS resources on behalf of the customer (By default we give administrator access to this IAM user since the CLI needs access to a variety of services for deployments)
+  - Creating an AWS Profile on your local system with reference to the accessKey and secretKey tied to the IAM user created in the above step. This AWS Profile could be given a custom name (be default we name it as 'default') and can be used for initializing any number of projects moving forward. Essentially the AWS profile defines which AWS account and region the AWS resources would be deployed to
+
+What happens behind the scenes?
+
+  - An IAM user is created in your AWS account with the appropriate policies required for the CLI to work with
+  - An AWS Profile is created on your local machine. If you check your ~/.aws/config file on your machine, you would observe a region associated with your profile name in the following format
+  ```
+  [profile amplifyprofile]
+  region=us-east-1
+  ```
+  - Similarly, in your ~/.aws/credentials file, you would see the accesskey and secretkey associated with your profile in the following format
+ ```
+[amplifyprofile]
+aws_access_key_id=AKIAI6N66xxxxxxxxxxxx
+aws_secret_access_key=4Nmtuxxxxxxxxxxxxxxxxxxxx
+```
+
+Whenever initializing a new project, the CLI would ask you to select an AWS profile and based on upon your selection all the corresponding resources are deployed to the region and IAM user tied to the selected profile.
+
+## amplify init
+```bash
+$ amplify init
+```
+
+The amplify init command is a one-time setup step for your Amplify powered cloud app which helps you with the following:
```

Comment:
I would change to this: "The amplify init command is a one-time initialization step for your Amplify powered cloud app. You run this once for each project (JavaScript, iOS, or Android) to connect your app with an AWS backend. This setup helps you with the following:"